tests/unit/api/BackbeatAPI.spec.js

```diff
@@ -49,6 +50,8 @@ describe('BackbeatAPI', () => {
             { url: '/_/metrics/crr/test', method: 'GET' },
             { url: '/_/metrics/crr/all/backlo', method: 'GET' },
             { url: '/_/metrics/crr/all/completionss', method: 'GET' },
+            { url: '/_/metrics/crr/all/failures', method: 'POST' },
+            { url: '/_/metrics/crr/all/fail', method: 'GET' },
```

Comment:
Is this line needed?

Reply:
no it isn't required, just tests that the route here should fail to parse. Should I remove?

Reply:
no that's fine